src/cmd/tools/m3db_tool/main/flagvar/file.go

```diff
@@ -0,0 +1,52 @@
+package flagvar
```

Comment:
This file can be removed. This code already exists https://github.com/m3db/m3/blob/a585b99328b57720a45a24946da33db9c0535497/src/x/config/configflag/flag.go#L102